tests/integration/extenders/ModelTest.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Tests\integration\extenders;
+
+use Carbon\Carbon;
+use Flarum\Discussion\Discussion;
+use Flarum\Extend;
+use Flarum\Group\Group;
+use Flarum\Post\Post;
+use Flarum\Tests\integration\RetrievesAuthorizedUsers;
+use Flarum\Tests\integration\TestCase;
+use Flarum\User\User;
+
+class ModelTest extends TestCase
+{
+    use RetrievesAuthorizedUsers;
+
+    protected function prepDb()
+    {
+        $this->prepareDatabase([
+            'users' => [
+                $this->adminUser(),
+                $this->normalUser(),
+            ]
+        ]);
+    }
+
+    protected function tearDown()
+    {
+        if (! is_null($discussion = Discussion::find(1))) {
+            $discussion->delete();
+        }
```

Comment:
You can achieve the same (completely independently from other tests that might run *before this one*) instead by adding the following line to the `prepareDatabase()` call:

~~~php
'discussions' => [],
~~~

Any table that is referenced in that call will be completely emptied before being filled with the provided rows - in this case, we would only empty the table.